src/provider/interface.ts

```diff
@@ -109,6 +109,14 @@ export abstract class ProviderInterface {
    */
   public abstract getTransaction(txHash: BigNumberish): Promise<GetTransactionResponse>;
 
+  /**
+   * Gets the transaction receipt from a tx hash or tx id.
+   *
+   * [Reference] (https://github.com/starkware-libs/cairo-lang/blob/master/src/starkware/starknet/services/api/feeder_gateway/feeder_gateway_client.py#L104-L111)
```

Comment:
Link should be https://github.com/starkware-libs/cairo-lang/blob/167b28bcd940fd25ea3816204fa882a0b0a49603/src/starkware/starknet/services/api/feeder_gateway/feeder_gateway_client.py#L183

And is it only tx hash?